src/localDB/dbConfig.ts

```diff
@@ -41,9 +41,46 @@ const dbPromise = openDB<ApplicationTrackerDB>("ApplicationTrackerDB", 1, {
 });
 
 // Add a new application
-export async function addApplication(application: Application) {
+export async function addApplication(application: ApplicationCreate) {
   const db = await dbPromise;
-  return db.add("applications", { id: uuid(), ...application });
+
+  const {
+    applicationDate,
+    company,
+    country,
+    coverLetter: clFromForm,
+    cv: cvFromForm,
+    location,
+    position,
+  } = application;
```

Comment:
this could be 
const {
    applicationDate,
    coverLetter: clFromForm,
    cv: cvFromForm,
...rest
  } = application;
  
  and later you can just do ...rest to give these props to db.add